diff --git a/node_modules/@stoplight/markdown-viewer/index.mjs b/node_modules/@stoplight/markdown-viewer/index.mjs
index d0a59fe..506110e 100644
--- a/node_modules/@stoplight/markdown-viewer/index.mjs
+++ b/node_modules/@stoplight/markdown-viewer/index.mjs
@@ -840,12 +840,13 @@ let sanitizationSchema = null;
 const buildSanitizationSchema = () => {
     if (!sanitizationSchema) {
         sanitizationSchema = deepmerge(defaultSchema, {
-            tagNames: ['tabs', 'tab', 'codegroup', 'button'],
+            tagNames: ['tabs', 'tab', 'codegroup', 'button', 'solved-icon'],
             attributes: {
-                '*': ['className', 'style', 'id'],
+                '*': ['className', 'class', 'style', 'id'],
                 code: ['title', 'lineNumbers', 'inline', 'highlightLines', 'lang', 'live', 'jsonSchema', 'http', 'resolved'],
                 img: ['bg', 'focus', 'inline', 'invertOnDark'],
                 blockquote: ['theme'],
+                'solved-icon': ['name'],
             },
         });
         sanitizationSchema.clobber = ['name'];
